CHANGELOG.md

```diff
@@ -8,6 +30,10 @@
 
 - support ruby 3.2 by renaming File.exists? to File.exist? [\#98](https://github.com/voxpupuli/librarian-puppet/pull/98) ([shamil](https://github.com/shamil))
 
+**Merged pull requests:**
+
+- Release 4.0.1 [\#99](https://github.com/voxpupuli/librarian-puppet/pull/99) ([ekohl](https://github.com/ekohl))
```

Comment:
I added the skip-changelog label, please regenerate.

Reply:
Bleh, only reviewed the PRs in 5.0.0, not the actual diff. Regenerated now.